pkg/apis/istio/v1alpha2/values_types.go

```diff
@@ -644,3 +646,19 @@ func FromInt(val int) IntOrStringForPB {
 func FromString(val string) IntOrStringForPB {
 	return IntOrStringForPB{intstr.FromString(val)}
 }
+
+// Validation checks
+func (t *PilotConfig) Validation(failOnMissingValidation bool, values *Values, icpls *IstioControlPlaneSpec) util.Errors {
+	var validationErrors util.Errors
+	validationErrors = util.AppendErr(validationErrors, fmt.Errorf("pilotconfig has not been yet implemented"))
```

Comment:
Probably just return nil so we can leave it as an example.

Reply:
done